app/domain/errors.rb

```diff
@@ -73,23 +73,23 @@ module AuthenticatorClass
 
       module Security
 
-        NotWhitelisted = ::Util::TrackableErrorClass.new(
-          msg: "'{0-authenticator-name}' is not whitelisted in CONJUR_AUTHENTICATORS",
+        AuthenticatorNotWhitelisted = ::Util::TrackableErrorClass.new(
+          msg: "'{0-authenticator-name}' is not enabled",
           code: "CONJ00004E"
         )
 
-        ServiceNotDefined = ::Util::TrackableErrorClass.new(
-          msg: "Webservice '{0-webservice-name}' is not defined in the Conjur policy",
+        WebserviceNotFound = ::Util::TrackableErrorClass.new(
+          msg: "Webservice '{0-webservice-name}' wasn't found",
           code: "CONJ00005E"
         )
 
-        UserNotAuthorizedInConjur = ::Util::TrackableErrorClass.new(
-          msg: "User '{0-user-name}' is not authorized in the Conjur policy",
+        RoleNotAuthorizedOnWebservice = ::Util::TrackableErrorClass.new(
+          msg: "'{0-role-name}' does not have 'authenticate' privilege on {1-service-name}",
           code: "CONJ00006E"
         )
 
-        UserNotDefinedInConjur = ::Util::TrackableErrorClass.new(
-          msg: "User '{0-user-name}' is not defined in Conjur",
+        RoleNotFound = ::Util::TrackableErrorClass.new(
```

Comment:
Maybe we should leave "In Conjur" to be clear

Reply:
I removed it because we don't want the word `Conjur` in our logs anyway